src/chi/components/expansion-panel/expansion-panel.scss

```diff
@@ -26,6 +26,7 @@ $transition-duration: 0.2s;
 
     & > .m-epanel__content {
       flex: 1;
+      word-break:break-word;
```

Comment:
@davidojedacl Can you add a space after the colon for code consistency?

Reply:
done